paymentsheet/src/main/java/com/stripe/android/common/model/CommonConfiguration.kt

```diff
@@ -25,6 +25,11 @@ internal data class CommonConfiguration(
     val externalPaymentMethods: List<String>,
     val cardBrandAcceptance: PaymentSheet.CardBrandAcceptance,
 ) : Parcelable {
+    private val isEKClientSecretValidRegexPattern = "^ek_[^_](.)+$"
+    private fun String.isEKClientSecretValid(): Boolean {
```

Comment:
nit: can you define this at the bottom of the file with the other extension functions?

Reply:
:+1: